libraries/js/src/credentials.ts

```diff
@@ -24,71 +20,34 @@ async function validateGraph(credential: SiwfResponseCredentialGraph): Promise<v
   }
 }
 
-export async function validateGeneralCredential(
-  credential: SiwfResponseCredential,
-  trustedIssuers: string[]
-): Promise<void> {
+export async function validateGeneralCredential(credential: SiwfResponseCredential): Promise<void> {
```

Comment:
This is basically checking only the valid date. I can remove it or add more verifications if desired.